docs/tactics.md

```diff
@@ -708,6 +708,24 @@ end
 ```
 and likewise for `to_rhs`.
 
+### conv
```

Comment:
This should be part of the section right above it, also titled `conv`. Maybe another bullet point in the list. Really, that whole section should probably be restructured, but also this whole file should be restructured, so both of those can wait for later.

Reply:
Good idea, I've done a bit of rearranging there now.